app/src/androidTest/java/com/github/orkest/View/auth/SignInTest.kt

```diff
@@ -174,4 +175,27 @@ class SignInTest {
         sharedPref.edit().clear().apply()
     }
 
+
+    /**
+     * Check if createNotificationChannel() works a
+     */
+    @Test
+    fun createNotificationChannelWorks() {
+        val context: Context = ApplicationProvider.getApplicationContext()
+        val notification = Notification(context,null)
+
+        notification.createNotificationChannel()
+    }
+
+    /**
+     * Check if sendNotification() works
+     */
```

Comment:
I think it's not necessary to add comments since the name of the tests speak for themselves and if i'm not mistaken we saw in sweng that we're allowed to have very long names for tests so they could be descriptive enough